core/group/service.go

```diff
@@ -99,13 +104,13 @@ func (s Service) AddUsers(ctx context.Context, groupIdOrSlug string, userIds []s
 		groupID = grp.ID
 	}
 
-	isAuthorized, err := s.relationService.CheckPermission(ctx, currentUser, namespace.DefinitionTeam, groupID, action.DefinitionManageTeam)
+	isAllowed, err := s.relationService.CheckPermission(ctx, currentUser, namespace.DefinitionTeam, groupID, action.DefinitionManageTeam)
 	if err != nil {
 		return []user.User{}, err
 	}
 
-	if !isAuthorized {
-		return []user.User{}, errors.Unauthorized
+	if !isAllowed {
+		return []user.User{}, errors.ErrForbidden
```

Comment:
why are we returning `[]user.User{}` ? We should be returning `nil` here.

Reply:
answered here
https://github.com/odpf/shield/pull/146#discussion_r948686861